src/Config/Converters/RestRuntimeOptionsConverterFactory.cs

```diff
@@ -47,6 +47,7 @@ public override void Write(Utf8JsonWriter writer, RestRuntimeOptions value, Json
             writer.WriteStartObject();
             writer.WriteBoolean("enabled", value.Enabled);
             writer.WriteString("path", value.Path);
+            writer.WriteBoolean("request-body-strict", value.RequestBodyStrict);
```

Comment:
To me it makes sense to write the value of this field to the config file as it might not be immediately evident for a user as to whether extra fields are allowed by the runtime. Specifically writing this property makes sense to me to make things clear. It is unlike other properties like rest.enabled which can be skipped (and is not skipped currently as you can see) and the user would still assume rest is enabled just because there is a rest section defined in the config.

Reply:
is it possible to add a test or validate what happens when config does not include the property `request-body-strict` ? this would help prevent regressions in backwards compatibility

Reply:
Added a test for the same via `ConfigurationTests.ValidateStrictModeAsDefaultForRestRequestBody`.